tests/get_function

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# Usage: get_function funcation_name filename
+# This script helps getting entire code snippet of a function
+# from the source code
+#
+start_line=`grep -n "^$1" $2|cut -d: -f1`
+sed -n "$start_line,$"p $2 | while IFS= read -r "line"
+do
+	echo "$line"
```

Comment:
This will interpret e.g. \ and $ as shell characters and modify the contents. That's the reason for your compile error.

Reply:
This is a shell script. I have not understood why this is happening. The same Makefile and script is working for me.

Reply:
Is there a way to pull in the libidn2-devel (libidn2-dev) library in the CI tests?

Reply:
The logs show that the CI is using ubuntu bionic. The correct library to install for adding EAI support would be `libidn2-0-dev`